addons/account/views/account_tax_report_views.xml

```diff
@@ -75,7 +75,6 @@
                             <group>
                                 <field name="tax_report_id" invisible="1"/>
                                 <field name="name"/>
-                                <field name="tax_report_line_id"/>
```

Comment:
I think this field should be invisible since it's a dependency of the `foreign_vat_fiscal_position_id` field

Reply:
Except it's not one of its dependencies ;p tax_report_country_id is, and it's a related field that's in the view :)